src/Components/Menu.module.css

```diff
@@ -1,49 +1,49 @@
 .navContainer {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    border: 1px solid black;
-    padding-left: 0.3rem;
-    padding-right: 0.3rem;
-    background-color: #FFF;
-    width: 100%;
-    flex-wrap: nowrap;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid black;
+  padding-left: 0.3rem;
+  padding-right: 0.3rem;
+  background-color: #fff;
+  width: 100%;
+  flex-wrap: nowrap;
 }
 
 .navElements {
-    display: flex;
-    gap: 0.8rem;
-    list-style: none;
-    padding-left: 0;
+  display: flex;
+  gap: 0.8rem;
+  list-style: none;
+  padding-left: 0;
```

Comment:
- [ ] I advise you to match the design as much as possible. The typography, color, and everything should match the given design. For example, the progress indicator does not match the design. Kindly see the image below.

![logo](https://user-images.githubusercontent.com/53124548/224488470-e2a167d6-fa7c-44b4-aa14-9272e49bed21.png)